helm-charts/basehub/values.yaml

```diff
@@ -244,6 +244,14 @@ jupyterhub:
                 matchLabels:
                   app.kubernetes.io/component: traefik
   hub:
+    config:
+      # Default options for CILogonOAuthenticator, for hubs where it is enabled
+      CILogonOAuthenticator:
+        scope:
+          - openid
+          - email
+          - org.cilogon.userinfo
```

Comment:
I really thought this would make hubs not using cilogon fail. But I guess, this just gets ignored, right?

Reply:
@GeorgianaElena yeah just gets ignored.